sycl/include/sycl/info/device_traits.def

```diff
@@ -195,6 +195,8 @@ __SYCL_PARAM_TRAITS_SPEC(device, ext_intel_mem_channel, bool,
                          PI_MEM_PROPERTIES_CHANNEL)
 __SYCL_PARAM_TRAITS_SPEC(device, ext_oneapi_bfloat16, bool,
                          PI_EXT_ONEAPI_DEVICE_INFO_BFLOAT16)
+__SYCL_PARAM_TRAITS_SPEC(device, ext_oneapi_bfloat16_math_functions, bool,
+                         PI_EXT_ONEAPI_DEVICE_INFO_BFLOAT16_MATH_FUNCTIONS)
```

Comment:
What is `ext_oneapi_bfloat16_math_functions` here?  Does this become a public API visible to applications, or is it in the `details` namespace?

Reply:
The symbols are internal between runtime and plugins. The name on Linux is: _ZNK4sycl3_V16device8get_infoINS0_4info6device19ext_oneapi_bfloat16EEENS0_6detail19is_device_info_descIT_E11return_typeEv

Reply:
When I demangle that string, it looks like the symbol is `sycl::info::device::ext_oneapi_bfloat16`.  Does that mean we are adding an info descriptor that is visible to user code via:

```
device d;
d.get_info<info::device::ext_oneapi_bfloat16>();
```